Bottom: fbe024f5d5d0f5821bcfa7e3023c12a8fc99c742
Top:    b2b0b84b4a45a35227f5805fd036a6623e3c33cf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-06 17:23:02 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 017ccd8..be96681 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -2669,10 +2669,9 @@ ghid_parse_arguments (int *argc, char ***argv)
   pixel_slop = 300;
 
   /* setup GL-context */
-  gport->glconfig = gdk_gl_config_new_by_mode (GDK_GL_MODE_RGB   |
-                                               GDK_GL_MODE_ALPHA |
+  gport->glconfig = gdk_gl_config_new_by_mode (GDK_GL_MODE_RGBA    |
                                                GDK_GL_MODE_STENCIL |
-//                                               GDK_GL_MODE_DEPTH |
+//                                               GDK_GL_MODE_DEPTH   |
                                                GDK_GL_MODE_DOUBLE);
   if (!gport->glconfig) {
     printf("Could not setup GL-context!\n");
